admin-dev/themes/default/template/controllers/stores/helpers/form/form.tpl

```diff
@@ -69,8 +69,37 @@
 			{foreach $fields_value.days as $k => $value}
 			<div class="form-group">
 				<label class="control-label col-lg-3">{$value}</label>
-				<div class="col-lg-9"><input type="text" size="25" name="hours_{$k}" value="{if isset($fields_value.hours[$k-1])}{$fields_value.hours[$k-1]|escape:'html':'UTF-8'}{/if}" /></div>
-			</div>
+        {if $languages|count > 1}
+          {foreach $languages as $language}
+            {if $languages|count > 1}
+              <div class="translatable-field lang-{$language.id_lang}" {if $language.id_lang != $defaultFormLanguage}style="display:none"{/if}>
+                <div class="col-lg-7">
+            {/if}
+                  <input type="text" size="25"
+                         name="hours_{$k}_{$language.id_lang}"
+                         value="{if isset($fields_value.hours[$language.id_lang][$k-1])}{$fields_value.hours[$language.id_lang][$k-1]|escape:'html':'UTF-8'}{/if}"/>
+            {if $languages|count > 1}
```

Comment:
Hello @PrestaEdit , I forgot to remove it when I added the check before.

Done thanks !